src/mudlet.cpp

```diff
@@ -416,15 +416,6 @@ mudlet::mudlet()
     timerAutologin->start( 1000 );
 
     connect(mpMainStatusBar, SIGNAL(messageChanged(QString)), this, SLOT(slot_statusBarMessageChanged(QString)));
-    // Do something with the QStatusBar just so we "use" it (for 15 seconds)...
```

Comment:
I think we have to _explicitly use/show the status bar __once___ - otherwise subsequent usages of `QAction::setStatusTip(`...`)` does _not work_ if the user does want the status bar - either permanently or as required.

🕧 Let me experiment and confirm (or deny) that...